File: microservices/company/src/main/java/com/illunex/invest/company/persistence/repository/PluginRepository.java
package com.illunex.invest.company.persistence.repository;

import com.illunex.invest.company.persistence.entity.Plugin;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface PluginRepository extends JpaRepository<Plugin, Long> {
    List<Plugin> findByCompanyCompanyIdxOrderByProductIdAsc(Long companyIdx);
}

File: microservices/shop/src/main/kotlin/com/illunex/invest/shop/persistence/repository/PluginRepository.kt
package com.illunex.invest.shop.persistence.repository

import com.illunex.invest.shop.persistence.entity.Plugin
import com.illunex.invest.shop.persistence.repository.custom.PluginCustomRepository
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface PluginRepository: JpaRepository<Plugin, Long>, PluginCustomRepository {
}